src/api/app/views/webui/image_templates/index.html.haml

```diff
@@ -3,13 +3,15 @@
     Choose a base template
 
   %fieldset.templateslist
+    - first_template = nil
     - @projects.each do |project|
       %dl.templateslist
         %dt
           = link_to(title_or_name(project), project_show_path(project))
-        - project.packages.each do |template|
+        - project.packages.each.with_index(1) do |template, index|
```

Comment:
Since we iterate through this it make sense to already load the packages in [the controller](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/controllers/webui/image_templates_controller.rb#L5).

See "Solution to N + 1 queries problem" section in http://guides.rubyonrails.org/active_record_querying.html#eager-loading-associations